_studio/shared/umc/codec/h264_dec/src/umc_h264_va_packer.cpp

```diff
@@ -862,8 +863,8 @@ Status PackerVA::QueryStreamOut(H264DecoderFrame* pFrame)
 
     int32_t const offset1 =  size * top;
     {
-        MFX_AUTO_LTRACE(MFX_TRACE_LEVEL_HOTSPOTS, "memcpy");
-        memcpy_s(dst + offset1, size, src, size);
+        MFX_AUTO_LTRACE(MFX_TRACE_LEVEL_HOTSPOTS, "std::copy");
```

Comment:
Isn't it useless here together with braces?

Reply:
good question. I just preserved tracing as it was, maybe there is no sense of it here. but then it should be another change to remove it